compiler/pipes/check-color.h

```diff
@@ -0,0 +1,373 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
```

Comment:
It's not a big deal, but I would use the 2021 copyright year for files that were created in 2021.